ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -80,8 +80,8 @@
     private static final By TELEPHONE_NUMBER_TEXT = By.cssSelector("#contactInfoContent div span.left-margin");
     private static final By RECENT_VISITS = By.cssSelector("visitbyencountertype > ul > li:nth-child(1) > a");
     private static final By PATIENT_GIVENNAME = By.cssSelector("#content div span.PersonName-givenName");
-
-    private static final By EDIT_REGISTRATION_INFORMATION = By.id("application.registrationapp.summary.editPatientLink");
+   
+    private static final By EDIT_REGISTRATION_INFORMATION = By.cssSelector("#application.registrationapp.summary.editPatientLink a");
```

Comment:
do you really mean https://css-tricks.com/whats-the-difference/#:~:text=A%20space%20in%20a%20CSS,because%20it%20has%20no%20space.?

an attribute selector that i suggested looks like By.cssSelector("a[id='application.registrationapp.summary.editPatientLink']")

Reply:
Resolved thanks.

Reply:
Hello @k-joseph  Have replied to all your comments looking forward to hear from you

Reply:
Resolved thanks cc @k-joseph